cfme/infrastructure/virtual_machines.py

```diff
@@ -132,24 +132,23 @@ def _nav_to_snapshot_mgmt(self):
                 sel.click(InfoBlock.element("Properties", "Snapshots"))
 
         def does_snapshot_exist(self):
+            name = self.name or self.description
```

Comment:
this method needs to use name to find snapshot if name was specified, same as it does now, and description is just a fallback if name wasn't specified (like with RHV, where snapshots have description, but no name)